helix-core/src/main/java/org/apache/helix/controller/stages/ResourceValidationStage.java

```diff
@@ -91,6 +87,29 @@ public void process(ClusterEvent event) throws Exception {
     }
   }
 
+  private void processManagementMode(ClusterEvent event, BaseControllerDataProvider cache)
```

Comment:
I am confused. This is general handle even function. If it is in management mode, then we throw exception. Would this affect the management pipeline handle event?

Reply:
No, because the management pipeline does not include this stage. This is data preprocess for default/task. Management does not run it.